doc/short_tutorial.rst

```diff
@@ -0,0 +1,169 @@
+
+Now everything should be ready to run the tutorial.
+
+Run the tutorial model
+---------------------
+
+A tutorial data kit was developed to facilitate exploring the model.
+The user can explore the majority of the model's functions on a local machine by running the tutorial, which uses fewer computational resources than the entire model does. 
+Currently, the tutorial case study refers to a microgrid in Nigeria whose coverage area is defined by a rectangle whose vertices have the following cooridnates:
+
+-	lon_max: 5.0998
+-	lon_min: 6.1700
+-	lat_min: 8.2356
+-	lat_max: 9.8012
+
+Before actually running the tool, it is always a good idea to check how it will look by using -dryrun or -n Snakemake option:
+
+.. code:: bash
+    .../pypsa-distribution (pypsa-earth) % snakemake -j 1 solve_all_networks --dryrun
+
+
+To run the whole modeling workflow you just need the following command:
+
+.. code:: bash
+    .../pypsa-distribution (pypsa-earth) % snakemake -j 1 solve_network
+
+.. note::
+    Before running these commands always make sure:
+    - you are in the correct folder ( i.e., the folder related to the project where the snakefile is located)
+    - that you have enabled the pypsa-earth environment
+
+.. TODO Explain settings of the tutorial case
+
+This command will trigger loading of the whole dataset needed to build the model for a tutorial case if both tutorial and retrieve_databundle flags are on. 
+The tutorial model  run simulation will take a while (about 20..50 minutes).
+
+
+Run a different case study
+--------------------------
+
+In this section you will find a small guide to learn how to configure the tool for your specific case study. 
+As an example, we will refer to the study of a microgrid in Kenya whose coverage area is defined by a rectangle with vertices in these coordinates: 
+
+- lon_max: 41.1141
+- lon_min: 41.1086
+- lat_min: -2.0596
+- lat_max: -2.0526
+To better understand this example, it might be helpful to read the section on configuration in the config.yaml file.
+
+.. note::
+    To find the coordinates of a specific study area, one functional way is to take advantage of the web application OpenStreetMap.
+
+First of all, it is always a good idea to start by running the default tutorial case study; this will allow you to see if the tool is working properly. 
+Furthermore, following the run of the tutorial case, you will see a "config.yaml" file appear in the folder. 
+This file conjointly with the config.pypsa-earth.yaml allows the user to specify the scenario they wish to analyze before running the algorithm.
+
+The following commands are the most relevant for setting up your personal case study:
+
+1. Configure the country: in the two configuration files ( config.yaml and config.pypsa-earth.yaml) you must enter under "country" the abbreviation of the country where your case study is located.In our exemple "KE"
+
+.. code:: yaml
+
+    countries: ["KE"]
```

Comment:
Attention: it would be better to keep the short_tutorial for only the tutorial cases. So, suggest here to keep only one of them, and for other region, please refer to the non-tutorial case.

Reply:
Perfect, I could remove this part ( from " run a different case study" ) and keep it to generate ( possibly later when the run for other configurations is properly executable) , another section in which to explain the simpler settings for running the run for another case study (?)

Reply:
Yes :) but please add a note to refer to the other section if the reader needs to do that